src/plugins/soundBoardLogger/utils.ts

```diff
@@ -50,6 +50,17 @@ export function getEmojiUrl(emoji) {
     return emoji.id ? `https://cdn.discordapp.com/emojis/${emoji.id}.png?size=32` : getURL(emoji.name);
 };
 
+const v1 = findByPropsLazy("amplitudeToPerceptual");
+const v2 = findByPropsLazy("getAmplitudinalSoundboardVolume");
+
+export const getSoundboardVolume = () => v1.amplitudeToPerceptual(v2.getAmplitudinalSoundboardVolume());
```

Comment:
Untested but should work
```suggestion
const { amplitudeToPerceptual } = findByPropsLazy("amplitudeToPerceptual");
const { getAmplitudinalSoundboardVolume } = findByPropsLazy("getAmplitudinalSoundboardVolume");

export const getSoundboardVolume = () => amplitudeToPerceptual(getAmplitudinalSoundboardVolume());
```

Reply:
trying to destructure those at the top level seems to cause a crash for reasons I do not understand

Reply:
Oh I see, I don't understand it either .-.

Reply:
errors do be funny what can I say

Reply:
destructuring the lazy find makes it non lazy, so it ends up trying to search for it before discord has loaded

Reply:
it's because it's lazily evaluated and isn't available on initialisation i guess

Reply:
lmao

Reply:
💀